code/_globals/lists/misc.dm

```diff
@@ -9,3 +9,19 @@ GLOBAL_LIST_EMPTY(wire_color_directory) // This is an associative list with the
 
 // Reference list for disposal sort junctions. Filled up by sorting junction's New()
 GLOBAL_LIST_EMPTY(tagger_locations)
+
+// A list of all the special byond lists that need to be handled different by vv
+GLOBAL_LIST_INIT(vv_special_lists, init_special_list_names())
+
+/proc/init_special_list_names()
+	var/list/output = list()
+	var/obj/sacrifice = new
+	for(var/varname in sacrifice.vars)
+		var/value = sacrifice.vars[varname]
+		if(!islist(value))
+			if(!isdatum(value) && hascall(value, "Cut"))
+				output += varname
+			continue
+		if(isnull(locate(REF(value))))
+			output += varname
+	return output
```

Comment:
i honestly really dislike doing this because we do have our own slightly more janky way of doing it, **and** because imo, doing this in the render loop for vv is better

Reply:
wdym render loop? it runs every vv UI call?